vmdb/app/presenters/menu/polymorph_item.rb

```diff
@@ -0,0 +1,7 @@
+module Menu
+  class PolymorphItem < Item
+    def name
+      super.call
```

Comment:
The menu is constructed once. There's a requirement to change the name any time you add/remove provider. Therefor you need to calculate the name each time the menu is displayed.